core/src/main/java/org/mobilitydata/gtfsvalidator/notice/ValidationNotice.java

```diff
@@ -29,5 +29,7 @@
 public abstract class ValidationNotice extends Notice {
   public ValidationNotice(Map<String, Object> context) {
     super(context);
+    // by default severityLevel is set to SeverityLevel.WARNING
+    this.setSeverityLevel(SeverityLevel.WARNING);
```

Comment:
Pass severity to the super() constructor.

Let's have this ERROR by default and leave WARNING to the next refactoring.